Jellyfin.Plugin.Bookshelf/Providers/OpfReader.cs

```diff
@@ -122,6 +124,20 @@ private Book CreateBookFromOpf()
             ReadStringInto("//dc:identifier[@opf:scheme='ISBN']", isbn => book.SetProviderId("ISBN", isbn));
             ReadStringInto("//dc:identifier[@opf:scheme='AMAZON']", amazon => book.SetProviderId("Amazon", amazon));
 
+            ReadStringInto("//dc:date", date =>
+            {
+                try
+                {
+                    var dateTime = DateTime.Parse(date, DateTimeFormatInfo.InvariantInfo);
```

Comment:
I would use `DateTime.TryParse` so you don't need to catch the exception

Reply:
Ah, nice to know that `DateTime.TryParse` is a thing. :)